x/stakeibc/keeper/host_zone.go

```diff
@@ -50,22 +52,33 @@ func (k Keeper) GetHostZone(ctx sdk.Context, chain_id string) (val types.HostZon
 	return val, true
 }
 
-// GetHostZoneFromDenom returns a hostZone from the relevant coin denom
-
+// GetHostZoneFromDenom returns a hostZone from the relevant coin denom (denom is ibc/...)
 func (k Keeper) GetHostZoneFromDenom(ctx sdk.Context, denom string) (val types.HostZone, found bool) {
 	store := prefix.NewStore(ctx.KVStore(k.storeKey), types.KeyPrefix(types.HostZoneKey))
 	iterator := sdk.KVStorePrefixIterator(store, []byte{})
+	var bval types.HostZone
 
 	defer iterator.Close()
 
+	// hash is the part of denom after ibc/
+	hash := strings.Join(strings.SplitN(denom, "ibc/", -1), "")
```

Comment:
should this be
```suggestion
	hash := strings.Join(strings.SplitN(denom, "ibc/", 1), "")
```
?

Reply:
they should result in the same output here, I just wanted to explicitly call out "go to the last section after ibc/" instead of "go to the first section after ibc/"